docs/operating-scylla/admin-tools/cassandra-stress.rst

```diff
@@ -94,7 +94,7 @@ CQL for the keyspace. Optional if the keyspace already exists:
 .. code-block:: cql
 
    keyspace_definition: |
-   CREATE KEYSPACE stresscql WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 3};
+   CREATE KEYSPACE stresscql WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 3};
```

Comment:
As this file relates to not only Scylla, but Cassandra as well, and also to cassandra-stress, which I'm not familiar with, I'd double check if that very line isn't actually used in our tests somewhere. Perhaps it's a line of code extracted from some function from the framework? (In which case we'd need to change it as well?)

Reply:
@fruch is it something you're familiar with?

Reply:
@fruch could you please comment? If not, we'll resolve this comment and proceed. (We can always fix that later).

Reply:
@KrzaQ this specific line describes what's the default test behaviour and as @annastuchlik pointed out, below in the FAQ section there's an explanation what to do in order to run tests with NetworkTopology.
For this reason I think the change in this specific line should be reverted.

Reply:
done